fix heading labels of CGUIDialogPeripheralSettings, CGUIDialogSmartPlaylistEditor and CGUIDialogVideoSettings

Montellese opened the pull request:
Some skinners noticed that a few dialogs didn't have a heading label set by default, see http://forum.kodi.tv/showthread.php?tid=214100&pid=1896191. These commits fix CGUIDialogPeripheralSettings, CGUIDialogSmartPlaylistEditor  and CGUIDialogVideoSettings based on the labels currently set by Confluence and removes those labels from Confluence.

The thread also mentions CGUIDialogVisualisationPresetList but that one has a label but it uses ID 3 because ID 2 is used for the list of presets (which obviously doesn't match the other dialogs but I don't want to break backwards compatibility).

Montellese committed:
CGUIDialogSmartPlaylistEditor: add default heading label

Montellese committed:
CGUIDialogPeripheralSettings: add default heading label

Montellese committed:
CGUIDialogVideoSettings: add default heading label

Montellese committed:
[confluence] remove default heading labels for DialogPeripheralSettings, SmartPlaylistEditor and VideoOSDSettings

ronie commented:
VideoOSDSettings.xml is also used for audio/subtitle settings.
the header is still hardcoded in confluence in that case.

Montellese commented:
@ronie: Ah I didn't realize that the same XML was used by two dialogs. But Confluence showed the same heading for both dialogs right?

ronie commented:
nope, it has two header labels in the xml file.

HitcherUK commented:
@ronie @Montellese Even though they both use the same dialog xml they do have different names, definitions and window IDs if that's any help.

`osdvideosettings	WINDOW_DIALOG_VIDEO_OSD_SETTINGS	10123	123	VideoOSDSettings.xml`
`osdaudiosettings	WINDOW_DIALOG_AUDIO_OSD_SETTINGS	10124	124	VideoOSDSettings.xml`

Nessus85100 commented:
@Montellese The new "Video Settings" string that you've add is already available here https://github.com/xbmc/xbmc/blob/master/language/English%20%28US%29/strings.po#L4976 although i have no idea why the "settings" word is not starts with capital "S" so can be used as a header label. Off-topic the same happens with several strings (eg 20343 "TV shows") which forces some skinners (like me) to assign their own labels for those strings.

Cheers
Nessus

ronie commented:
@Nessus85100 we always only capatilize the first word.
(not saying this is good or bad, but just that it's done consistent throughout the language file).

Nessus85100 commented:
@Ronie Completely wrong if you ask me, which is not happening in other languages. Here is the "TV shows" string translated properly with capital letters in both words in Greek: https://github.com/xbmc/xbmc/blob/master/language/Greek/strings.po#L8682.

Cheers
Nessus

Montellese committed:
CGUIDialogAudioSubtitleSettings: add default heading label

Montellese commented:
Updated to use the existing "Video settings" label and also fixed `CGUIDialogAudioSubtitleSettings` and removed the second header label from Confluence's `VideoOSDSettings.xml`.

@ronie: Let me know if something else needs to be done.

I'll do the header label ID alignment of the other dialog in a separate PR.

ronie commented:
looks ok to me. thx!

Montellese commented:
jenkins build this please

Montellese commented:
I had to stop the WIN32 jenkins build because it was already running for 2.5 hours whereas it usually takes 25-30 minutes. But I tested WIN32 compiliation myself.

kib commented:
don't forget to update the forum thread! :+1:
